content/en/docs/components/katib/katib-config.md

```diff
@@ -0,0 +1,211 @@
+
+Katib config is the Kubernetes
+[Config Map](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/) that contains information about:
+
+1. Current [metrics collectors](/docs/components/katib/experiment/#metrics-collector) (`key = metrics-collector-sidecar`)
+1. Current [algorithms](/docs/components/katib/experiment/#search-algorithms-in-detail) (suggestions) (`key = suggestion`).
+
+Katib Config Map must be deployed in
+[`KATIB_CORE_NAMESPACE`](/docs/components/katib/env-variables/#katib-controller)
+namespace with `katib-config` name. Katib controller parses Katib config when you submit Experiment.
+
+You can edit this Config Map even after deploying Katib.
+
+If you deploy Katib in Kubeflow namespace, to edit Katib config run this:
+
+`kubectl edit configMap katib-config -n kubeflow`
+
+## Metrics Collector Sidecar settings
+
+These settings are related to Katib metrics collectors, where:
+
+- key = `metrics-collector-sidecar`
+- value = corresponding JSON settings for each metrics collector kind.
+
+Example for the `File` metrics collector with all settings:
+
+```json
+metrics-collector-sidecar: |-
+{
+  "File": {
+    "image": "docker.io/kubeflowkatib/file-metrics-collector",
+    "imagePullPolicy": "Always",
+    "resources": {
+      "requests": {
+        "memory": "200Mi",
+        "cpu": "250m",
+        "ephemeral-storage": "200Mi"
+      },
+      "limits": {
+        "memory": "1Gi",
+        "cpu": "500m",
+        "ephemeral-storage": "2Gi"
+      }
+    }
+  },
+  ...
+}
+```
+
+All of these settings except **`image`** can be omitted. If you don't specify any other settings, default value is set.
+
+1. `image` - Docker image for the `File` metrics collector's container.
+
+   **Must be specified**.
+
+1. `imagePullPolicy` - [Image pull policy](https://kubernetes.io/docs/concepts/configuration/overview/#container-images) for the `File` metrics collector's container.
+
+   Default value is `IfNotPresent`.
+
+1. `resources` - [Resources](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container)
+   for the `File` metrics collector's container. In the above example you can see how to specify
+   `limits` and `requests`. Currently, you can specify only `memory`, `cpu` and `ephemeral-storage` resource.
+
+   Default values for the `requests` are:
+
+   - `memory = 10Mi`.
+   - `cpu = 50m`.
+   - `ephemeral-storage = 500Mi`.
+
+   Default values for the `limits` are:
+
+   - `memory = 100Mi`.
+   - `cpu = 500m`.
+   - `ephemeral-storage = 5Gi`.
+
+## Suggestion settings
+
+These settings are related to Katib suggestions, where:
+
+- key = `suggestion`
+- value = corresponding JSON settings for each algorithm name.
+
+If you want to use new algorithm, you must update Katib config with the new suggestion.
+
+Example for the `random` algorithm with all settings:
```

Comment:
Suggestion:

```suggestion
If you want to use a new algorithm, you need to update the Katib config. For example, using a `random` algorithm with all settings looks as follows:
```

Reply:
There's a Markdown that described the `random` algorithm - maybe we should it link it at the first mention in this doc? Up to you.

Reply:
If you want to use a new algorithm, you need to update the Katib config.

Reply:
> There's a Markdown that described the `random` algorithm - maybe we should it link it at the first mention in this doc? Up to you.

@8bitmp3 Do you mean this doc: https://deploy-preview-2312--competent-brattain-de2d6d.netlify.app/docs/components/katib/experiment/#random-search ?